react/components/Autocomplete/components/CustomListItem/CustomListItem.tsx

```diff
@@ -48,6 +48,9 @@ export class CustomListItem extends React.Component<CustomListItemProps> {
                     skuId: sku && sku.itemId,
                   }}
                 >
+                  <span className="dib t-small c-muted-2">
```

Comment:
Could you add some margin-right here?
The ListPrice and SellingPrice are stuck together

Reply:
of course, it's for now

Reply:
I added a display block and decreased the font?

![image](https://user-images.githubusercontent.com/38081030/87585163-05fbf500-c6b5-11ea-9b6a-79664e8348a9.png)